docs/third-party-cdn.md

```diff
@@ -0,0 +1,60 @@
+# Third Party CDN Usage
+
+In some situations it may be required to use your own CDN to serve the Altis powered website or service. Below are some examples that may necessitate this:
+
+- Many web properties are served from a single domain name, whereby you need to control routing. A sub-section of the URL namespace may route to Altis, in which case it's not possible to route all traffic to Altis' CDN for a single domain name.
+- There may be compliance requirements or company-wide mandated use of a specific CDN vendor.
+
+In these cases, Altis Cloud can be configured to serve traffic to your CDN via an origin-pull setup. Altis does not support an origin-push CDN setup. Reach out to Altis Cloud Support for assistance in third party CDN integration as it requires specific manual work by the Altis team.
+
+## CDN Configuration
+
+Altis Cloud will provide origin DNS names for each Cloud environment (`development`, `staging` and `production`) to set as the Origin-Pull upstream for you CDN. For example, this could be `development.example.altis.cloud`. The CDN should be configured with the logic described below, in whatever format or system the CDN uses. See below for special routes `/tachyon/*` and `/uploads/*`.
+
+### Cookies
+
+All cookies matching the pattern `wordpress_*`, `wp-*`, `wp_*`, `comment_*` and `hm_*` should be forwarded to the Altis Cloud origin.
+
+### URL Query Parameters
+
+All URL query parameters should be forwarded to the Altis Cloud origin.
+
+### HTTP Headers
+
+The following HTTP headers should be forwarded: `Host`, `Origin`, `Authorization`.
+
+### Caching Configuration
+
+The CDN caching rules should be set to respect the origin's `Cache-Control` HTTP header. For example, if an origin response includes the header `Cache-Control: max-age=300`, the CDN should cache the response for 300 seconds.
+
+The following type of requests should be set to skip the CDN cache entirely:
+
+- All `POST` or `DELETE` requests.
+- Any requests with cookies as described above.
+- Any requests with the `Authentication` HTTP header.
+- Any response with the `Cache-Control: no-cache` HTTP header.
+
+The cache key should be based on the following string: `{http method}:{protocol}:{host}:{path}:{query_params}`.
```

Comment:
Is it up to them to ignore certain query strings the way we do for batcache? Not sure if it's worth adding a note that they may want to exclude common ones like `utm_campaign` etc

Reply:
We don't support this on our CDN so I don't think so no.